src/components/Dropdown/Dropdown.tsx

```diff
@@ -104,11 +112,28 @@ export class Dropdown extends React.Component<DropdownProps, State> {
     }
 
     const { onChange } = this.props;
-    this.setState({ selectedOption });
+    this.setState({
+      selectedOption,
+      isOpen: false,
+      ariaActivedescendant: null,
+    });
     onChange &&
       onChange(this.props.options.find(({ id }) => selectedOption.id === id));
   };
 
+  private readonly onOptionHover = (option: DropdownOptionProps & IDOMid) => {
+    const ariaActivedescendant = option ? option._DOMid : null;
```

Comment:
I feel like I'm missing something.
no one will ever pass `_DOMid` to the option, because it doesn't exist in the `DropdownOptionProps` interface.
Why not just use the option's `id` property?

Reply:
this ensures us that the option will have a domId ->https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/dropdown-content/DropdownContent.tsx#L150
You can also checkout the onOptionHover function there.

Reply:
I used it that way because that what I saw they are doing in ui-core and also this is the function's signature.